stats.sql

```diff
@@ -0,0 +1,21 @@
+-- -*- mode: sql; product: postgres -*-
+
+-- Create the statistics view
+-- This is what the statistics API endpoint must expose, formated as JSON.
+CREATE VIEW statistics AS
+  SELECT name, users, maxusers, data
+    FROM hosts
+    JOIN (
+            SELECT COUNT(*)
+                AS users, host
+              FROM passwd
+          GROUP BY host
+         )
+      AS count
+      ON (count.host = hosts.name)
+;
+
+
+-- Create a role for the statistics endpoint
+CREATE ROLE "statistics";
```

Comment:
Ah. looks like we never created an anon role (nor an authenticator role).
I guess that should go in a separate PR.
So for now just remove the `GRANT` all-together and we can merge it.